README.md

```diff
@@ -244,71 +271,75 @@ AwsSsmEnv.load!(
 
 #### other
 
-`AwsSsmEnv::NamingStrategy`を実装したクラスのインスタンス、もしくは`parse_name`メソッドを持つ  
-インスタンスを指定した場合はそのインスタンスをそのまま利用する。
+If you specify an instance of a class that implements `AwsSsmEnv::NamingStrategy` in` fetch`,
+or an instance with a `parse_name` method, use that instance as is.
+
 
 ### path: [String]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合は必須となる。  
-パラメータを取得するパス階層を指定する。  
-下の例では`/myapp/web/production`直下のパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Specify the path hierarchy for acquiring parameters.  
+In the example below, the parameter immediately under `/myapp/web/production` is acquired.
 
 ```ruby
 AwsSsmEnv.load(path: '/myapp/web/production')
 ```
 
 #### recursive: [Boolean]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合に利用する。  
-指定したパス階層以下のパラメータをすべて取得する。  
-下の例では`/myapp/web/production`以下すべてのパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Acquires all parameters below the specified path hierarchy.  
+In the following example, all parameters below `/myapp/web/production` are acquired.
 
 ```ruby
 AwsSsmEnv.load(path: '/myapp/web/production', recursive: true)
 ```
 
 ### begins_with: [String, Array<String>]
 
-`fetch`に`:begins_with`を指定した場合は必須となる。  
-取得するパラメータ名のプレフィクスを指定する。配列で複数指定することも可能(OR条件となる)。  
-下の例では`myapp.web.production`で始まる名前のパラメータが取得される。
+It is required if `:begins_with` is specified in `fetch`.  
+Specify the prefix of the parameter name to be acquired.
```

Comment:
The same two points with above are also applicable here.

For example:

> Required if `:begins_with` is specified in `fetch`.
You can specify the prefix of the parameter name to be acquired by this option.